src/pybind/mgr/ansible/run-tox.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# run from ./ or from ../
+: ${MGR_ANSIBLE_VIRTUALENV:=/tmp/mgr-ansible-virtualenv}
+: ${WITH_PYTHON2:=ON}
+: ${WITH_PYTHON3:=ON}
+: ${CEPH_BUILD_DIR:=$PWD/.tox}
+test -d ansible && cd ansible
```

Comment:
What if this directory does not exist? Shouldn't you create it?

Reply:
I just have followed the same approach present in dashboard and insights, although i have to confess that I have not studied with detail the build process for the test environments.

In any case, I have check that tests run smoothly in local environment even If you remove the folder "/ tmp / mgr-ansible-virtualenv".
I know that this folder is created in the build process, you can build it manually with:
`make mgr-ansible-test-venv`

So i suppose that the folder is needed in the automatic tests executed in Jenkins. But is just a suspicion. It would be nice to know the explanation of this from somebody with a deeper knowledgment of the build/tests environment. Until i know for what is used i prefer leave it as is.